charts/gardener/gardenlet/charts/runtime/templates/configmap-componentconfig.yaml

```diff
@@ -0,0 +1,125 @@
+{{- if .Values.global.gardenlet.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: gardenlet-configmap
+  namespace: garden
+  labels:
+    app: gardener
+    role: gardenlet
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+data:
+  config.yaml: |
+    ---
+    apiVersion: gardenlet.config.gardener.cloud/v1alpha1
+    kind: GardenletConfiguration
+    gardenClientConnection:
+      acceptContentTypes: {{ required ".Values.global.gardenlet.config.gardenClientConnection.acceptContentTypes is required" .Values.global.gardenlet.config.gardenClientConnection.acceptContentTypes }}
+      contentType: {{ required ".Values.global.gardenlet.config.gardenClientConnection.contentType is required" .Values.global.gardenlet.config.gardenClientConnection.contentType }}
+      qps: {{ required ".Values.global.gardenlet.config.gardenClientConnection.qps is required" .Values.global.gardenlet.config.gardenClientConnection.qps }}
+      burst: {{ required ".Values.global.gardenlet.config.gardenClientConnection.burst is required" .Values.global.gardenlet.config.gardenClientConnection.burst }}
+      {{- if .Values.global.gardenlet.config.gardenClientConnection.kubeconfig }}
+      kubeconfig: {{ .Values.global.gardenlet.config.gardenClientConnection.kubeconfig }}
+      {{- end }}
+      {{- if .Values.global.gardenlet.kubeconfig }}
+      kubeconfig: /etc/gardenlet/kubeconfig/kubeconfig
```

Comment:
That's how it was today as well: https://github.com/gardener/gardener/blob/master/charts/gardener/charts/runtime/templates/controller-manager/configmap-componentconfig.yaml#L23-L28
But I'll add a `elif` to prevent this, thanks.